src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/AmendCourtBookingService.kt

```diff
@@ -58,6 +58,11 @@ class AmendCourtBookingService(
       videoUrl = request.videoLinkUrl
       this.amendedBy = amendedBy.username
       amendedTime = now()
+      notesForPrisoners = request.notesForPrisoners
+
+      if (amendedBy is ExternalUser) {
+        notesForStaff = request.notesForStaff
```

Comment:
We will only use private notes when requested by an external user.

Reply:
Is this right? ExternalUser == court/probation - its right they can add only private/staff notes, but I think prison users can add either?